drivers/hrldas/NoahmpGroundwaterInitMod.F90

```diff
@@ -124,14 +117,16 @@ subroutine NoahmpGroundwaterInitMain(grid, NoahmpIO)
          enddo
 
       endif
-    enddo !NITER
 
-#if (EM_CORE == 1)
-#ifdef DM_PARALLEL
-#     include "HALO_EM_HYDRO_NOAHMP.inc"
-#endif
+#ifdef MPP_LAND
+      rcount=float(ncount)
+      call sum_real1(rcount)

```

Comment:
is there any issue here when using different precision? since sum_real1 is a single precision function?

Reply:
why not just write a new sum_integer function?